docs/sphinx/user_guide/feature/view.rst

```diff
@@ -82,6 +82,55 @@ accesses array entries with unit stride. The loop::
 
 access array entries with stride N :subscript:`n` * N :subscript:`(n-1)` * ... * N :subscript:`(j+1)`.
 
+MultiView
+^^^^^^^^^^^^^^^^
+
+A ``RAJA::MultiView`` object wraps an array-of-pointers,
+or a pointer-to-pointers, whereas a ``RAJA::View`` wraps a single
+pointer or array. This allows a single ``RAJA::Layout`` to be applied to
+multiple arrays internal to the MultiView, allowing multiple arrays to share indexing
+arithmetic when their access patterns are the same.
+
+The instantiation of a MultiView works exactly like a standard View,
+except that it takes an array-of-pointers. In the following example, a MultiView
+applies a 1-D layout of length 4 to 2 internal arrays in ``myarr``::
+
+  int a1[4] = {5,6,7,8}; // Arrays of the same size, which will become internal to the MultiView.
+  int a2[4] = {9,10,11,12};
+  int * myarr[2]; // Array-of-pointers which will be passed into MultiView.
+  myarr[0] = a1;
+  myarr[1] = a2;
+  // This MultiView applies a 1-D layout of length 4 to each internal array in myarr.
+  RAJA::MultiView< int, RAJA::Layout<1> > MView(myarr, 4);
+
+The default MultiView accesses internal arrays via the 0th index of the MultiView::
+
+  MView( 0, 4 ); // accesses the 4th index of the 0th internal array a1, returns value of 8

```

Comment:
Does the "internal" qualifier add anything?

Reply:
I wanted to distinguish between the array-of-pointers, and the arrays within the array-of-pointers (internal arrays).